src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -233,7 +224,7 @@ class SourcesTree extends Component<Props, State> {
     const menuOptions = [];
 
     if (!isDirectory(item)) {
-      const source = item.contents.get("url");
+      const source = item.contents.url;
```

Comment:
Maybe name this `sourceURL` and change the corresponding `copyToTheClipboard` as this is what we are actually getting here.